### PR TITLE
New Page AnimesOnline

### PR DESCRIPTION
**Pull request by Martins-74**
Added a new page animesonline

### CODE REVIEW

File: src/pages/AnimesOnline/main.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  isOverviewPage(url) {
+    return ['anime', 'animes-dublado'].includes(utils.urlPart(url, 3));
+  },
+  sync: {
+    getTitle(url) {
+      return j.$(j.$('.breadcrumb li')[2]).text().replace('- Dublado', '').trim();
+    },
+    getIdentifier(url) {
+      return j.$(j.$('.breadcrumb li')[2]).text().trim().replace('- Dublado', '');
+    },
```
Comment (lolamtisch) [inline src/pages/AnimesOnline/main.ts:43]: duplicated code. One could just call `return AnimesOnline.sync.getTitle(url);`

File: src/pages/AnimesOnline/main.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      return j.$(j.$('.playerbox .controls a')[2]).attr('href');
+    },
+  },
+  overview: {
+    getTitle(url) {
+      return j.$(j.$('.breadcrumb li')[2]).text().replace('- Dublado', '').trim();
+    },
+    getIdentifier(url) {
+      return j.$(j.$('.breadcrumb li')[2]).text().replace('- Dublado', '').trim();
+    },
```
Comment (lolamtisch) [inline src/pages/AnimesOnline/main.ts:60]: More duplicated code. You can just call the functions of the sync section

File: src/pages/AnimesOnline/main.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+  $('.player.current').removeClass('current');
+  $('#tab-1').addClass('current');
+
+  $($('video')[0]).attr('src', currentPlayer === 1 ? originalSrc : tempSrc ?? originalSrc);
+};
+
+export const AnimesOnline: pageInterface = {
+  name: 'Animes Online',
+  domain: 'animesonline.in',
```
Comment (lolamtisch) [inline src/pages/AnimesOnline/main.ts:28]: Please use `https://animesonline.in`

File: src/pages/AnimesOnline/main.ts [hunk 4]
```diff
@@ -0,0 +1,106 @@
+import { pageInterface } from '../pageInterface';
+
+let originalSrc;
+const changeSrc = (currentPlayer, currentTries = 0) => {
+  const tempSrc = $(`#tab-${currentPlayer} video`).attr('src');
+
+  if (tempSrc === undefined) {
+    if (currentTries >= 100) {
+      alert('MAL-SYNC: Cant find target src video, please refresh the page and try again!');
```
Comment (lolamtisch) [inline src/pages/AnimesOnline/main.ts:9]: Alerts are a bit hard. Please use 
```
utils.flashm(`MAL-SYNC: Cant find target src video, please refresh the page and try again!`, {
    error: true,
    type: 'error',
  });
```

**Review by lolamtisch — CHANGES_REQUESTED** [review]

**Comment by Martins-74** [comment]
Requested changes done

**Pull request merged**